server/src/instant/system_catalog.clj

```diff
@@ -275,7 +276,7 @@
               :unique? true
               :index? true)
    (make-attr "$files" "path"
-              :unique? false
+              :unique? true
```

Comment:
This won't be applied automatically, we'll have to create an indexing job for it. I can do that after we deploy the changes.